applications/tari_collectibles/web-app/src/Dashboard.js

```diff
@@ -46,15 +47,13 @@ class DashboardContent extends React.Component {
 
     try {
       let outputs = await binding.command_assets_list_registered_assets(0, 100);
+      console.log(outputs);
```

Comment:
Take out?

Reply:
Can stay for now. These pages are by no means final